src/PProf.jl

```diff
@@ -163,7 +163,7 @@ function pprof(data::Union{Nothing, Vector{UInt}} = nothing,
             # End of sample
             value = [
                 1,                   # events
-                length(location_id), # stack_depth
+                60000000             # CPU ns
```

Comment:
what does this mean?

Reply:
Hm yeah, I think I found I needed to put some number here to get it to show up in DD, but this is just a bogus value. Not sure what it's supposed to be… Maybe it's supposed to correspond to the sample period? i.e. if you're taking a stack sample every 100ms, this is supposed to be 100ms (in ns)?

Reply:
ah, yes, that's right. i think that is exactly what it's supposed to be. it's supposed to be the value matching the value type:
```
        ValueType!("cpu", "nanoseconds")
```
👍 

🤔 i think you should be able to ask `Profile` what it's current sample rate is, so that this can be an accurate number.
```julia
help?> Profile.init()
  init(; n::Integer, delay::Real)

  Configure the delay between backtraces (measured in seconds), and the number n of instruction pointers that may be stored per thread. Each instruction
  pointer corresponds to a single line of code; backtraces generally consist of a long list of instruction pointers. Note that 6 spaces for instruction
  pointers per backtrace are used to store metadata and two NULL end markers. Current settings can be obtained by calling this function with no arguments,
  and each can be set independently using keywords or in the order (n, delay).

  │ Julia 1.8
  │
  │  As of Julia 1.8, this function allocates space for n instruction pointers per thread being profiled. Previously this was n total.

julia> Profile.init()
(10000000, 0.001)

julia> Profile.init()[2]
0.001
```